data/jsonSeed.js

```diff
@@ -0,0 +1,72 @@
+const fs = require('fs');
+const faker = require('faker');
+const agents = require('./agents.json')
+
+// generateAgents(1000000);
+generateListings(10000000);
+
+function generateAgents(number) {
+  let writableAgents = fs.createWriteStream('./data/agents.json', { 'flags': 'a' });
+
+  let data = '';
+  for (let i = 1; i <= number; i++) {
+    let obj = {
+      id: i,
+      name: faker.name.findName(),
+      company: faker.company.companyName(),
+      reviews: faker.random.number(100),
+      recentSales: faker.random.number(80),
```

Comment:
I would highly recommend using Math.random instead of faker.random.number wherever possible due to efficiency gains (~30% faster)

Reply:
i don't think we're trying to optimize the generation of the data so I'm not too worried about it, but thanks!